UPGRADE-3.4.md

```diff
@@ -68,6 +68,9 @@ Process
 
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
+   
+ * The `Symfony\Component\Process\Process` __construct `cwd` argument will throw 
+   a deprecation warning on `run` if the folder does not exist.
```

Comment:
You need to rebase your chases on top of `3.4` and then you can change the base branch by clicking the "Edit" button on the top right of your PR.